.gitignore

```diff
@@ -156,3 +156,4 @@ docs/source/04_user_guide/source/.ipynb
 tests/template/fake_project/
 
 default.profraw
+package-lock.json
```

Comment:
This is generated from running `make devserver` which builds a version of the docs you can read locally - we should exclude it from accidentally being committed